ocs_ci/ocs/jenkins.py

```diff
@@ -349,6 +352,36 @@ def export_builds_results_to_googlesheet(
              f"test_run_name:{utils.get_testrun_name()}"], 2
         )
 
+    def get_nodes(self, node_type=constants.WORKER_MACHINE, num_of_nodes=1):
```

Comment:
The final selected nodes is a list of nodes_worker_name - nodes_app_name.
Can we change the function name to a appropriate name,?